labs-ios-starter/Login/View Controllers/LoginViewController.swift

```diff
@@ -62,17 +62,17 @@ class LoginViewController: DefaultViewController {
                 self.presentedViewController == nil else { return }
                 
             if exists {
-                self.performSegue(withIdentifier: "ShowDetailProfileList", sender: nil)
+                self.performSegue(withIdentifier: .getSegueIdentifier(identifier: .showDetailProfileList), sender: nil)
```

Comment:
Would be nice to shorten this up. I wonder if you can put the enum directly inside the string extension

Reply:
we could just do as you suggested in a previous comment and use the enum with Raw values. I was trying to eliminate us having to remember to store the raw value separately. Both approaches have their unique drawbacks and benefits. We could also make static properties on String using the convenience method to capitalize names. I.e. `static var someIdentifier: String = .getSegueIdentifier(.someIdentifier)` the we'd have `.someIdentifier` to use. But it's an extra step, so might as well use the enum lol

Reply:
Just thinking about this more, another downside to just using the rawValue is you lose access to calling it directly (You would have to call `.Identifier.someIdentifier.rawValue`) - that seems as long

Reply:
I could (should?) make `identifier:` unnamed

Reply:
Right. Hmm. I’m going to clone and play with this for a second.